### DISCUSSION TITLE
Are there eslint file pattern base config equivalent in clippy?

### DISCUSSION THREAD

**Discussion by loynoir** [rust-lang/rust-clippy#13397] [Q&A]
Are there eslint file pattern base config equivalent in clippy?

eslint.config.js

```ts
  {
    files: ['**/*.test.js', '**/*.wip.js'],
    ignores: [],
    rules: {
      '@foo/foo': 'off',
      '@bar/bar': 'error'
    }
  },
```

Want to 

- enable some rules for source code.

- but not  `xxx_test.rs`

For example

```toml
disallowed-methods = [
    { path = "core::result::Result::unwrap", reason = "prefer ?" },
```

But want to allow `unwrap` on all `xxx_test.rs`.

**Comment by flip1995** [comment]
No, this is not possible. `test`s also only get linted if you lint with `cargo clippy --all-targets` or `--tests`. 
Or you can allow the lint on top of your test files.

For enabling/disabling lint, this might be a possible extension for the `[lints]` table in the `Cargo.toml` file, so a `cargo`, not a `clippy` feature.

For `unwrap` we also have the `unwrap_used` lint that you can enable.
